test/test_lock.py

```diff
@@ -73,7 +73,7 @@ def wait_for_pid(timer):
 
     def close_lock_holder(self):
         try:
-            self.other_process.communicate("whatever")
+            self.other_process.communicate("whatever".encode('utf-8'))
```

Comment:
BTW: this: `cd src; grep -R -i 'encode("utf' * | wc -l` gives me 2 :-).

Reply:
Changing this and other instances where it makes sense to unicode literals (`u''` syntax)...

Reply:
Er... nevermind. That doesn't really make sense and it broke things. Coffee hasn't kicked in yet. :coffee: